packages/pwa-kit-react-sdk/src/ssr/universal/components/storefront-preview/index.jsx

```diff
@@ -9,13 +9,15 @@ import React, {useEffect} from 'react'
 import PropTypes from 'prop-types'
 import {Helmet} from 'react-helmet'
 import {detectStorefrontPreview, getClientScript} from './utils'
-
+const defaultGetToken = () => {
+    return window.localStorage.getItem('token')
+}
 /**
  *
  * @param {boolean} enabled - flag to turn on/off Storefront Preview feature
- * @param {{function():string}} getToken - a STOREFRONT_PREVIEW customised function that fetches token of storefront
+ * @param {function(): string} getToken - a STOREFRONT_PREVIEW customised function that fetches token of storefront
```

Comment:
```suggestion
 * @param {function(): string | Promise<string>} getToken - A method that returns the access token for the current user
```
1. `STOREFRONT_PREVIEW` is an internal detail - I don't think we ever actually expose that interface to the customer directly!
2. We also support returning `Promise<string.`. (Not currently, but the change will land soon on the Runtime Admin side, so we can update the doc now.)